ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Hello! Thanks for clicking that shiny new issue button.
+
+We want you to get help when you need it and opening an issue here will probably just slow you down, so could you do us a favor? Go check README.md and curators.md to make sure this is the most appropriate place for what you're about to post. We have much faster ways of solving individual problems, and they're linked to there.
+
+...
+
+Still here? Great! Just a little bit of work will make sure we hear your voice and solve your problem in the long term. Could you please fill out the sections below? Just follow the directions in each one, and delete these instructions when you're done.
+
+We may or may not respond to your issue, but we appreciate you taking the time to open it! Issues usually have to "bake" for a while so things can be solved the right way. Check out that talk linked in README.md for why.
```

Comment:
I see your point, but it's a deviation from the error message catalog. Is that important?